eZ/Publish/Core/REST/Server/Input/Parser/Limitation/LimitationParser.php

```diff
@@ -0,0 +1,96 @@
+<?php
+/**
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Publish\Core\REST\Server\Input\Parser\Limitation;
+
+use eZ\Publish\Core\REST\Common\Input\BaseParser;
+use eZ\Publish\Core\REST\Common\Input\ParsingDispatcher;
+use eZ\Publish\Core\REST\Common\Exceptions;
+use eZ\Publish\API\Repository\Values;
+
+/**
+ * Parses limitation objects from input.
+ *
+ * The class expects
+ */
+class LimitationParser extends BaseParser
```

Comment:
Naming can be improved here.

This is a special parser, that has a behaviour of its own:
- It builds the class name it is given
- It uses a named route parameter to parse the href into the limitation value(s)